Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -2760,11 +2806,16 @@ def get_events(client: Client, args: dict) -> Union[CommandResults, str]:
             'EventID': event.get('id'),
         })
 
+        for c in column_to_display:
+            if c in event:
+                contents[-1][c] = event.get(c)
```

Comment:
```suggestion
        for c in set(column_to_display).intersection(event.keys()):
             contents[-1][c] = event[c]
```

Same reasoning as above, but it's up to you if you want to implement it.